### PR TITLE
llama cake

### PR DESCRIPTION
**Pull request by KaitlinJKhosravi**
magical

### CODE REVIEW

File: _posts/2018-01-27-llama-cake.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "some food"
+date: 2018-01-27
+author: me
+permalink: ./some-food/
```
Comment (mhrivnak) [inline _posts/2018-01-27-llama-cake.md]: Please choose a unique link instead, perhaps ``llama-cake``

File: _posts/2018-01-27-llama-cake.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: "some food"
```
Comment (mhrivnak) [inline _posts/2018-01-27-llama-cake.md]: And a unique title would be a good idea too!

**Comment by KaitlinJKhosravi** [comment]
@mhrivnak did the updates! Can you review again? The world needs to know about the llama cake!!! :cowboy_hat_face:

**Comment by mhrivnak** [comment]
Thanks!!!

**Pull request merged**